app/main.py

```diff
@@ -1,8 +1,47 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
```

Comment:
Place each argument on a new line so that there would be no arguments on the same line with opening and closing parenthesis to improve code style and readability.